README.md

```diff
@@ -27,4 +27,265 @@
 - `odom_frame`
 - `map_frame`
 - `publish_tf`
-- `transform_tolerance`
\ No newline at end of file
+- `transform_tolerance`
+
+
+# Running openvslam on ros2
+
+## Installation using docker
+
+- `Instructions for SocketViewer` 
+- `Instructions for PangolinViewer` - `Comming soon`
+
+# Instructions for SocketViewer
+`Dockerfile.socket` can be used for easy installation. This chapter provides instructions on building and running examples with SocketViewer support using Docker.
+
+## Building Docker Images
+
+### Docker Image of OpenVSLAM
+
+### Step 1: Clone the required github repository:
+
+If you are going to use non RGBD setup then exceute following in your terminal.
+
+    git clone https://github.com/OpenVSLAM-Community/openvslam_ros.git
+
+If you are going to use RGBD setup then exceute following in your terminal.
+
+    git clone -b wip/approximate-time-rgbd https://github.com/OpenVSLAM-Community/openvslam_ros.git
+
+In case of openvslam using rosbag file, then replace `Dockerfile.socket` with `Dockerfile.socket-rosbag`.
```

Comment:
It seems that the old description still exists.

Reply:
Is it also required to delete below lines?
If you are going to use RGBD setup then exceute following in your terminal.

    git clone -b wip/approximate-time-rgbd https://github.com/OpenVSLAM-Community/openvslam_ros.git